src/applications/claims-status/containers/ClaimPage.jsx

```diff
@@ -3,21 +3,27 @@ import { connect } from 'react-redux';
 import { Outlet, useNavigate, useParams } from 'react-router-dom-v5-compat';
 import PropTypes from 'prop-types';
 
-import { getClaim as getClaimAction } from '../actions';
+import {
+  getClaim as getClaimAction,
+  clearClaim as clearClaimAction,
+} from '../actions';
 
-export function ClaimPage({ getClaim }) {
+export function ClaimPage({ getClaim, clearClaim }) {
   const navigate = useNavigate();
   const params = useParams();
 
   useEffect(() => {
     getClaim(params.id, navigate);
+    // Reset claim and loading state on dismount.
+    return () => clearClaim();
```

Comment:
Are there any possible side-effects here? Is there anywhere in the app where we _want_ to use previously loaded claim data outside of this component (and its children in the router)?

Reply:
At this time I can't think of any reason we would want to use the claim details data outside of the pages that are already reusing it currently